ViAn/Project/Analysis/analysismeta.cpp

```diff
@@ -0,0 +1,85 @@
+#include "analysismeta.h"
+#include <QJsonObject>
+#include <QString>
+
+/**
+ * @brief AnalysisMeta::AnalysisMeta
+ * @param analysis
+ */
+AnalysisMeta::AnalysisMeta(Analysis &analysis)
+{
+    m_name = analysis.name.toStdString();
+    m_full_path = analysis.m_full_path;
+    std::vector<POI> pois = analysis.POIs;
+    for (auto it = pois.begin(); it != pois.end(); ++it) {
+        POI poi = *it;
+        std::pair<int,int> pair = std::make_pair(poi.start_frame, poi.end_frame);
+        m_poi_intervals.push_back(pair);
+    }
+    std::cout << "in analysis constructor" << m_full_path << std::endl;
```

Comment:
we don't want unnecessary prints in master. clean some of the functions in this file,